pkg/kf/commands/install/kf/kf.go

```diff
@@ -85,6 +95,51 @@ func Install(ctx context.Context, containerRegistry string) error {
 	return nil
 }
 
+func waitForKfDeployments(ctx context.Context) error {
+	ctx, cancel := context.WithTimeout(ctx, time.Minute)
+	defer cancel()
+
+	for _, deploymentName := range []string{"controller", "webhook"} {
+		Logf(ctx, "waiting for %s deployment to be available...", deploymentName)
+		err := wait.ExponentialBackoff(
+			wait.Backoff{
```

Comment:
If I understand correctly this polls once per second and doesn't increase the time per iteration. We might want to do something like 5 seconds with a 1.5 factor.